### DISCUSSION TITLE
Hello , I need help to start your project

### DISCUSSION THREAD

**Discussion by Ophelie22** [thuongtruong1009/metasave#21] [Q&A]
Hello , i tried to install your project  with docker. I make .env with this information. But I see only  Hello world when i go on localhost: 3000.

I'm a novice with Vue and Mpngo maybe you can explaine me what i have to do? Thanks a lot.

APP_PORT=3000
APP_URL=http://localhost:${APP_PORT}

NODE_ENV=development

ORIGIN='http://localhost:3000,http://localhost:3001'

# Database
DB_NAME=metasave
MONGO_URL=mongodb://localhost:27017/metasave
ATLAS_URL= a the root of the project but

**Comment by thuongtruong1009** [comment]
Oh, hello @Ophelie22 

I am very glad that you are interested in my project. Based on the information you just shared, I suggest the following direction for you:

* There are 2 options for you, 1 is run with **Docker**, 2 is run **local**

**a) Here is run with Docker (no setup, no configs)**
**Step 0**: Create a new account and download Docker
**Step 1**: Open 2 terminals, one at root and the other at a folder named client.
**Step 2**: At the terminal at root, type docker-compose up
**Step 3**: After waiting for the container to finish, switch to the terminal at the folder named client, type npm install (install dependencies) , then npm run dev (start Vue)

**b) Here is run with local**
**Step 1**: Create an account at Postgres homepage. Then download PostgresSQl, and PostgresAdmin
**Step 2**: Download MongoDB, and Mongo Compass
**Step 3**: Open 2 terminals, one at root and the other at a folder named client.
**Step 4**: At the terminal at root, type npm install, and then npm run dev
**Step 5**: At the terminal at the client name folder, type npm install (install dependencies) , then npm run dev (start Vue)

* As you can see, in development target mode, **MONGO_URL=mongodb://localhost:27017/metasave** will be the URL of MongoDB locally, if you want to deploy it to cloud, then you have to create project at Mongo Atlas, then paste your project url at field **ATLAS_URL=<your_atlast_url>**

Ok, based on these two ways, you can go to http:localhost://3000 to enjoy the product just started

Thank you for following and interested in my project. Hope to have your support in future projects
